tests/rule_based_profiler/parameter_builder/test_parameter_container.py

```diff
@@ -51,9 +51,7 @@ def test_get_fully_qualified_parameter_names(
     }
 
     expected_fully_qualified_parameter_names: List[str] = [
-        "$variables.my_int",
-        "$variables.my_float",
-        "$variables.my_string",
+        "$variables",
```

Comment:
This is also very important `"$variables"` is now accessible at the base-level. It was `"$variables.my_int"` before